### DISCUSSION TITLE
Show us your GLaBIOS PCs

### DISCUSSION THREAD

**Discussion by 640-KB** [640-KB/GLaBIOS#26] [Show and Share]
Share pictures, specs and details about your GLaBIOS PC!

**Comment by 640-KB** [comment]
**I'll start!**

This is my IBM PC 5150 (16-64K early version), running GLaBIOS 0.2.3.

![20230112_200311](https://user-images.githubusercontent.com/23486433/212218431-c60231c0-87c9-46d8-9121-a94e4564b89a.jpg)

Firing up some old classics!

![20230112_200236](https://user-images.githubusercontent.com/23486433/212218429-a2f95463-79fd-4a37-bfba-ad12fe85520c.jpg)

**Comment by TzOk83** [comment]
This is my NEC V20 powered Micro 8088, running GLaBIOS 0.2.4:
![2023-04-13 23 14 48](https://user-images.githubusercontent.com/34161432/231886020-9740a78f-17ce-470a-89de-9e326c882bfa.jpg)
![2023-04-13 19 25 38](https://user-images.githubusercontent.com/34161432/231886040-71f556d4-b282-4604-b114-0be9b140757a.jpg)
![2023-01-14 12 05 33](https://user-images.githubusercontent.com/34161432/231886224-f37044bd-8b22-4d20-99fa-10e126f74f03.jpg)

**Reply by 640-KB** [reply]
Love that frame chassis there - that's beautiful minimalism!  

What's going on with the spaces between the LPT and the FDC on the post screen?  Is there another option ROM in there that's doing that?

**Comment by TzOk83** [comment]
I use it as a demo unit in Computer Architecture classes (at least it was my official justification for building it), so I've decided to keep it open. The frame was needed to keep cards parallel to each other to ensure good contact in the slots.

Yes, that's the option ROM (JOINCOM ELECTRONIC CORP. HD FDD) to be blamed for additional spaces on the POST screen.

**Reply by 640-KB** [reply]
Makes sense - just wanted to make sure it wasn't something I messed up. :)

Did you do all the boards yourself too?  I like that slot bracket - 3D printed?

**Reply by TzOk83** [reply]
The boards (ISA backplane, Micro8088, and XT-CF-Mini) were ordered in JLCPCB and PCB-Way, but I have assembled them. The VGA, sound card, and multi-IO cards are some old generic ISA cards.

The back frame and card bracket are 3D printed in gray E-HT PLA.

**Reply by 640-KB** [reply]
Very nice work.  The gray bracket looks really slick.  That settles it, next build I do is going to be an open frame!

**Comment by csoren** [comment]
This is my NuXT V1 modern XT. I'm a little space constrained and generally try to keep things as compact as possible, it lives in a low-profile Antec VSK2000-U3 cabinet.

It has a V20 CPU, 8087 FPU and a 3.5" floppy drive (the 5.25" I have wouldn't fit, sadly)

The four ISA slots are populated with some modern cards, (left to right) a 2 MB EMS card, RTC, OPL-3 Adlib clone, PS/2 mouse to serial. The NuXT V2 has the RTC and PS/2 mouse card integrated, but I have found these for the V1. The bigger cards *just* fit in the case, but they were not designed as low-profile cards. I have recently gotten more into 3D printing so I can design some proper brackets for the cards.

The keyboard is WASD V2 with Cherry MX Blue switches. I also have a Model M, but as I said, space-constrained. This one gives me the same feeling and it looks pretty good with the case.

![_1000759](https://user-images.githubusercontent.com/195392/232190448-73449694-21f4-4443-91bc-6c4e759ac2e2.JPG)
![_1000764](https://user-images.githubusercontent.com/195392/232190474-c181a8f3-cfb3-4b6d-a651-b92ff8d06d04.JPG)

**Reply by csoren** [reply]
Oh, right, before I had the RTC card I used a WiFi-modem on the serial port to get the time from a time server on the internet. It worked fine (and wasn't slow) but it was a little clunky to also have the modem connected.

**Reply by 640-KB** [reply]
Wow, that's beautiful!  Love the way the keyboard matches the case so nicely both in size and color.  What a fortunate fit for those cards in that case.  I had been looking for a low-profile case too and just couldn't find the perfect one that would fit standard height cards while still being small enough but looks like you found it!

Nicely done!

**Comment by explit28** [comment]
My both GlaBIOS Systems:

- Micro8088 with NEC V20 9,5 MHz CPU
- Juko ST-12 Test Station with Fujitsu 8088 8 MHz CPU

![Micro8088_Station_Front_View](https://github.com/user-attachments/assets/d6980740-e751-4091-9c0c-2c0e9052555d)
![Micro8088_Mainboard_13](https://github.com/user-attachments/assets/c88a07ec-f1b4-4548-81c9-58db12c97515)
![Juko_ST-12_Test_Station](https://github.com/user-attachments/assets/212de00c-6e16-4d92-867a-3820d0b37f5e)
![Juko_Baby_XT_GLaBIOS_SVGA](https://github.com/user-attachments/assets/4b116e96-f34b-481e-95bf-4bc803d876a4)
